### DISCUSSION TITLE
Stable-Release

### DISCUSSION THREAD

**Discussion by SoumadeepChoudhury** [SoumadeepChoudhury/Ahens-Remindering-Todo#11] [Announcements]
# This is the stable release of ART. Even there could be few bugs which will be fixed based on users feedback here in Discussions page.

<hr /><em>This discussion was created from the release <a href='https://github.com/SoumadeepChoudhury/Ahens-Remindering-Todo/releases/tag/stable'>Stable-Release</a>.</em>

**Comment by imanmay2** [comment]
I AM NOT GETTING THE UPDATES TILL NOW...
SO, I THINK IT IS A BUG 🐛😭🐛🐛..
TRY TO FIX IT

**Reply by SoumadeepChoudhury** [reply]
### _Thanks For the Feedback. We will work on it soon !!!!_
